airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -0,0 +1,60 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/source-acceptance-tests.md)
+# for more information about how to configure these tests
+connector_image: airbyte/source-amazon-ads:dev
+tests:
+  spec:
+    - spec_path: "integration_tests/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog_products.json"
+      empty_streams:
+        [
+          "sponsored_product_negative_keywords",
+          "sponsored_product_targetings",
+          "sponsored_product_ads",
+        ]
+      expect_records:
+        path: "integration_tests/expected_produts_stream.txt"
+        extra_fields: no
+        exact_order: no
+        extra_records: yes
+
+    - config_path: "secrets/config.json"
```

Comment:
btw, you can ommit config_path, it has a default value

Reply:
Explicit is better than implicit

Reply:
true, but not in the case when you repeat it over and over. Beautiful is better then ugly :)